PollBuddy-Server/frontend/src/App.js

```diff
@@ -59,7 +59,7 @@ export default class App extends React.Component {
 
         <MDBContainer id="wrapper">
 
-          <Header title={this.state.pageTitle} userInfo={this.state.userInfo} />
+          <Header />
```

Comment:
I think we'll need the userInfo passed like that for the dropdown to show relevant fields (when that works) but I could be wrong.